aicsimageio/tests/readers/extra_readers/test_ome_tiled_tiff_reader.py

```diff
@@ -46,19 +46,6 @@
             [f"C:{i}" for i in range(10)],  # This is the actual metadata
             (None, None, None),
         ),
-        pytest.param(
-            # This is the same file as the first file, but it is not tiled
-            # This should throw and error since it is not tiled
-            "s_1_t_1_c_1_z_1.ome.tiff",
-            None,
-            None,
-            None,
-            None,
-            None,
-            None,
-            None,
-            marks=pytest.mark.xfail(raises=exceptions.UnsupportedFileFormatError),
-        ),
```

Comment:
just wondering, why remove these two?

Reply:
Reading these files with latest libraries does not raise Error